lib/config/index.js

```diff
@@ -76,6 +76,11 @@ function getDefaultConfig() {
         throw new GeminiError('No config found');
     }
 
+    if (!/\.yml$/.test(configFile)) {
+        // Add absolute path to config (otherwise we will look for a package named `.gemini.js`)
```

Comment:
Initially I couldn't think of a good way to do this but I think I have a solution using `process.cwd` and `process.cd`